app/src/main/java/com/example/adhirajdatar/mdbsocials/Social.java

```diff
@@ -0,0 +1,28 @@
+package com.example.adhirajdatar.mdbsocials;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by datarsd1 on 2/23/17.
+ */
+
+public class Social {
+
+    String title, host, description, date, firebaseLocation, id, hostuid;
+    ArrayList<String> interested;
+    Integer numInterested;
+
+    public Social(String t, String h, String desc, String d, String fbl, String id, Integer numInterested, String hostuid) {
```

Comment:
I know no one ever sees these temporary parameters, but please try to be more descriptive than h